html-css/override-extend-variables/intro.md

```diff
@@ -1,7 +1,7 @@
-<strong>In this module, you will learn how to override and extend PatternFly css in the context of your application to accomodate your custom designs.</strong>
```

Comment:
Update the json file so that it has actual titles for the steps rather than step 1, step 2 etc

<img width="142" alt="Screen Shot 2019-12-04 at 5 02 07 PM" src="https://user-images.githubusercontent.com/20118816/70185479-dc5b0400-16b7-11ea-880c-eaa4dff5a7d0.png">